src/Magnum/Platform/Sdl2Application.h

```diff
@@ -1015,6 +1053,10 @@ class Sdl2Application {
         typedef Containers::EnumSet<Flag> Flags;
         CORRADE_ENUMSET_FRIEND_OPERATORS(Flags)
 
+        #ifndef CORRADE_TARGET_EMSCRIPTEN
+        SDL_Cursor* _cursors[14];
```

Comment:
```suggestion
        SDL_Cursor* _cursors[14]{};
```

(I got a crash otherwise. Same for GLFW.)